### PR TITLE
Fetch Role (/me) refactoring

### PR DESCRIPTION
**Pull request by odeimaiz**
- Do not request /me twice
- Set Role when validating user token
- Do not store user email in Permissions class
- check EntryPoint added to resources

related to #880

### CODE REVIEW

**Comment by ignapas** [comment]
Sorry Odei, I didn't see this one until Pedro sent it to me yesterday. I am on it.

**Comment by odeimaiz** [comment]
> The user profile, that contains the user role and the login email, is cached after the first call to /me using the `Resources` class and it is accessible from anywhere in `osparc.store.Store.getInstance().getProfile()`, properties `role` and `login`. Maybe we are having duplicates and unnecessary extra calls here.

There is a bit of a mess regarding role and login. I agree we should centralize them all in the profile object cause at the moment these information is in
- osparc.auth.Data
- osparc.data.Permissions
- osparc.store.Store
We could have another PR for doing a proper refactoring.

> And also, what are the new `checkEP` resources for?

It's just an entry point for testing/diagnostic. It returns an echo message or a failure so that the correct callback can be implemented

File: services/web/client/source/class/osparc/data/Resources.js [hunk 1]
```diff
@@ -323,6 +323,23 @@ qx.Class.define("osparc.data.Resources", {
             url: statics.API + "/activity/status"
           }
         }
+      },
+
+      /*
+       * Test/Diagnonstic entrypoint
+       */
+      checkEP: {
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/data/Resources.js]: do you really used this? If so, POST needs a body to comply with openAPI

Comment (odeimaiz) [reply services/web/client/source/class/osparc/data/Resources.js]: We try to make all REST calls through this class and whether a body is required or not is not defined here.

**Pull request merged**